ci/scripts/r_docker_configure.sh

```diff
@@ -39,6 +39,16 @@ if [ "$RHUB_PLATFORM" = "linux-x86_64-fedora-clang" ]; then
   rm -rf $(${R_BIN} RHOME)/etc/Makeconf.bak
 fi
 
+# Special hacking to try to reproduce quirks on centos using non-default build
+# tooling.
+if [[ "$R_TAG" == *centos* && "$DEVTOOLSET_VERSION" > 0 ]]; then
```

Comment:
We won't set this except on a centos job, so you probably don't need to check R_TAG (which I believe isn't defined at this point anyway, this script gets called in a dockerfile, and R_TAG isn't set in the dockerfile).

```suggestion
if [ "$DEVTOOLSET_VERSION" > 0 ]; then
```